Formula/s/swiftlint.rb

```diff
@@ -2,22 +2,18 @@ class Swiftlint < Formula
   desc "Tool to enforce Swift style and conventions"
   homepage "https://github.com/realm/SwiftLint"
   url "https://github.com/realm/SwiftLint.git",
-      tag:      "0.53.0",
-      revision: "6d2e58271ebc14c37bf76d7c9f4082cc15bad718"
+      tag:      "0.55.1",
+      revision: "b515723b16eba33f15c4677ee65f3fef2ce8c255"
   license "MIT"
   head "https://github.com/realm/SwiftLint.git", branch: "main"
 
   bottle do
-    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "240ccda9de55d948d0c635798079074099bfcb73ffda41428900fdc748aeea7b"
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "7b7ceb7896c6833965cc4eac9001255d8adde6c5432045d5a8ab6aea8a9e81d9"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "78c2a4c3f4a2f6847b484527b0f0f916da71e3ee29e49890fd44b63fe7b38e26"
-    sha256 cellar: :any_skip_relocation, sonoma:         "abdca78dd8a8bd268053b3be195fe891bb74aef5502ab3a6b871ae0c6bb04540"
-    sha256 cellar: :any_skip_relocation, ventura:        "be711c707bf3b49fa0dd6e2ae576b309aad620f9b56a2c6e7b1ac5cf35cf652a"
-    sha256 cellar: :any_skip_relocation, monterey:       "13487d68a971dbe035019364e19d70641af2a18c06e52925d238685b384a7979"
-    sha256                               x86_64_linux:   "fbbc56fccfcfcd34564feb7325567e2ff3638d3c609396a5c4aa13311c7b26e0"
+    sha256 cellar: :any_skip_relocation, arm64_sonoma: "25a8cbd68645815070428bd121bb38ae6d46d06726dee56fcabc0d578ed8f879"
+    sha256 cellar: :any_skip_relocation, sonoma:       "39c0c24936b847f924034aa405d55db7ac40a875d6b79c632281a0d0ade4ab42"
+    sha256                               x86_64_linux: "551c6e567e00130f816eeef8b08db9013d594216e315957550ed3f8c098984e8"
   end
 
-  depends_on xcode: ["14.0", :build]
+  depends_on xcode: ["15.3", :build]
```

Comment:
@SimplyDanny Since you bump the Xcode version to 15.3 (which is only available in the latest macOS 14+ Sonoma https://xcodereleases.com/), it makes it impossible to install the latest version of `swiftlint` via homebrew on macOS 13

Given that installing the previous versions of the formula in homebrew isn't properly supported operation, it leads to the impossibility to install `swiftlint` on macOS less than 14 via homebrew

More context: my GitHub Actions builds started failing on macOS 13 because they cannot install `swiftlint` anymore https://github.com/nikitabobko/AeroSpace/actions/runs/9120846693/job/25084905714

> swiftlint: A full installation of Xcode.app 15.3 is required to compile
> this software. Installing just the Command Line Tools is not sufficient.
> Error: swiftlint: An unsatisfied requirement failed this build.
> 
> Xcode 15.3 cannot be installed on macOS 13.
> You must upgrade your version of macOS.
> Error: swiftlint: An unsatisfied requirement failed this build.

What I also tried:
- I tried to search for the macOS mention in [0.54.0](https://github.com/realm/SwiftLint/releases/tag/0.54.0) and [0.55.0](https://github.com/realm/SwiftLint/releases/tag/0.55.0) release notes. They don't mention anything about minimum required macOS (was the bump of minimum required macOS intended?)
- [swiftlint README.md](https://github.com/realm/SwiftLint/blob/main/README.md) doesn't mention anything about minimum macOS version

Do I miss something?

Reply:
The reason is that SwiftLint has required Swift 5.9 to build since version 0.54.0. At least Swift 5.9 is available from Xcode 15.3 only.

But thinking about this, it seems like the Xcode build dependency is actually not necessary anymore. There is a `swift` formula in Homebrew as well. It should be sufficient to build SwiftLint. I'll check if that would work.

Reply:
#171993 has landed, so there should be binaries for older macOS versions now.

Reply:
I confirm that the problem is fixed. Thank you!